.eslintrc.js

```diff
@@ -554,12 +557,13 @@ const appRules = {
   'no-restricted-imports': [
     'error',
     {
-      paths: [
+      patterns: [
         {
-          name: 'enzyme',
-          message:
-            'Please import from `sentry-test/enzyme` instead. See: https://github.com/getsentry/frontend-handbook#undefined-theme-properties-in-tests for more information',
+          group: ['sentry/components/devtoolbar/*'],
```

Comment:
pulled this up from an override below. The default is to not import from this folder... this this folder has an override so this rule is not applied to files inside the folder.